### PR TITLE
[Issue #461] Fix updates lost when clicking on a different venture

### PR DESCRIPTION
**Pull request by peterphan1996**
## Status
**READY**

## Issue links
https://github.com/venturemark/webclient/issues/461

## Description
Fix updates lost when clicking on a different venture

### CODE REVIEW

File: src/component/header/index.tsx
Comment (tfussell) [inline src/component/header/index.tsx:49]: I tend to prefer wrapping with `Link` because it means you can see where it will take you before you click on it (in the status bar). Also you can right click and select "Open in a new tab". What do you think?

**Review by tfussell — CHANGES_REQUESTED** [review]
The general approach makes sense. I can approve once the merge conflicts are fixed.

File: src/component/header/index.tsx
Comment (peterphan1996) [reply src/component/header/index.tsx:49]: I see what you mean, I'll follow what you did in your previous PR then. Thanks

**Review by tfussell — APPROVED** [review]

**Pull request merged**
